Refactored rev1 page to clean up

kdmurray opened the pull request:
- rev1 page is now reading from a json file with all the image slugs, comments and builder data
 - thumbnail files have been added

kdmurray committed:
Refactored rev1 page to clean up

kdmurray committed:
Restoring the "Want to Contribute?" card; accidentally removed.

kdmurray committed:
Added images; fixed a few bugs

ctypeh commented on data/rev1.json:
```diff
@@ -0,0 +1,506 @@
+    }, 
+    {
+        "id": 33,
+        "imageslug": "2020-08-07_18.26.58",
```
This image is incorrect. Should be 2020-08-09_12.53.38

ctypeh commented on data/rev1.json:
```diff
@@ -0,0 +1,506 @@
+    {
+        "id": 33,
+        "imageslug": "2020-08-07_18.26.58",
+        "title": "SFC Conservatory",
```
Also, would prefer to see this written out as Sunflower City

ctypeh commented on rev1.html:
```diff
@@ -56,560 +53,26 @@ <h2>Early Days</h2>
         </div>
 
+        <script>
+        
```
Empty script tag?

ctypeh commented on data/rev1.json:
```diff
@@ -0,0 +1,506 @@
+        "title": "SFC Conservatory",
+        "desc": "Conservatory at Sunflower City",
+        "builder": "",
+        "date": "August 7th, 2020"
```
Date incorrect as well

ctypeh reviewed (changes requested):
New pictures added are great additions, thanks for gathering them.
Change to using json/js to build cards and the thumbnails are fantastic, thank you so much for implementing that. This is much neater than my attempt

One issue–
Nearly the same one I experienced. At screen resolutions where only a single column is visible (767px and below), I am seeing certain cards and images be much wider than others and disrupting the flow. I will look at it today as I have time

ctypeh committed:
Fixed incorrect SFC Conservatory data

ctypeh commented:
Also, more a stylistic note, I'm not sure every image should display the date since a lot of them are just whenever I happened to log in and take the picture and not really fixed in time. I might want to go back through and remove a few to reduce clutter. What do you think?

kdmurray committed:
Removed empty script tag.

kdmurray committed:
Merge branch 'master' of https://github.com/kdmurray/legos.digital

ctypeh committed:
Made page maximum possible width

see images more clearly
gallery-view is better
ensures any solutions we make for card image thumbnail sizes are future-proof

ctypeh committed:
Fixed card rendering size issue

ctypeh commented:
Fixed issues with card rendering, all other nitpicks resolved
